conan/api/model/list.py

```diff
@@ -276,8 +276,29 @@ def refs(self):
                 result[recipe] = rrev_dict
         return result
 
+    def items(self) -> dict[RecipeReference, dict[PkgReference, dict]]:
```

Comment:
Type is not really correct, this is not a dict, but a view.

The idea is that this iterator gives a simpler ``{RecipeReference: {PkgReference: {Binary info}}`` view